short_answer/static/html/short_answer.html

```diff
@@ -4,12 +4,17 @@
     <div class="description">
         <p>{{ description }}</p>
     </div>
-    <textarea class="short-answer" name="short-answer">{{ answer }}</textarea>
+    <textarea class="short-answer" name="short-answer"
+              {% if grades_published %}disabled{% endif %}>{{ answer }}</textarea>
```

Comment:
Indentation is weird, change it to one of the following:
```
<textarea class="short-answer" name="short-answer" {% if grades_published %}disabled{% endif %}>
    {{answer}}
</textarea>
```
or
```
<textarea class="short-answer" name="short-answer" {% if grades_published %}disabled{% endif %}>{{answer}}</textarea>
```

based on your preference.